operator/pkg/client/client_proxy.go

```diff
@@ -0,0 +1,90 @@
+package client
+
+import (
+	"context"
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/rest"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const restMappingErr = "client proxy failed to get resource mapping for"
+
+func NewRuntimeClient(config *rest.Config, mapper meta.RESTMapper) (client.Client, error) {
```

Comment:
This method is unnecessary, the original is already exposed

Reply:
This implementation was changed as discussed. Please check again.